src/storage/mod.rs

```diff
@@ -491,17 +482,14 @@ pub struct Storage {
 
 impl Storage {
     pub fn from_engine(engine: Box<Engine>, config: &Config) -> Result<Storage> {
-        let (tx, rx) = mpsc::sync_channel(config.scheduler_notify_capacity);
-        let sendch = SyncSendCh::new(tx, "kv-storage");
-
         info!("storage {:?} started.", engine);
+
         Ok(Storage {
             engine: engine,
-            sendch: sendch,
-            handle: Arc::new(Mutex::new(StorageHandle {
-                handle: None,
-                receiver: Some(rx),
-            })),
+            worker: Builder::new("storage-command-scheduler")
```

Comment:
s/storage-command-scheduler/storage-scheduler

Reply:
I noticed this problem, the scheduler cpu metric will not  work if using `storage-command-scheduler`.